### PR TITLE
a word about IPython 4.x

### PR DESCRIPTION
**Pull request by Carreau**
@willingc, could you review my phrasing ?

### CODE REVIEW

File: news.rst [hunk 1]
```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
 reviewer. We thank Packt Publishing for donating a portion of the proceeds
 from this book to support IPython's development.
 
+IPython 4.0
+-----------
+
+IPython 4.0 was release on August 11, 2015. This release correspond to the
```
Comment (takluyver) [inline news.rst]: was released

Comment (takluyver) [inline news.rst]: corresponds

File: news.rst [hunk 2]
```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
...
+separation of IPython (as of 3.x) into multiple components, most of which are
+now under the Jupyter organisation. See `the jupyter blog post
```
Comment (takluyver) [inline news.rst]: Capital J for jupyter

File: news.rst [hunk 3]
```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
...
+<https://blog.jupyter.org/2015/04/15/the-big-split/`_ for further explanations.
```
Comment (takluyver) [inline news.rst]: explanation

(explanations is valid, but sounds a bit odd - it's usually used as a mass noun, like 'flour')

File: news.rst [hunk 4]
```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
...
+
+IPython 4.0 contain only the codebase for the features of IPython terminal
```
Comment (takluyver) [inline news.rst]: contains

**Comment by takluyver** [comment]
I've done a quick proof reading pass - I'm happy to do more on it tomorrow if @willingc hasn't got round to it.

File: news.rst [hunk 5]
```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
...
+itself. Extra shims and warning have been added to ease the transition of
```
Comment (willingc) [inline news.rst]: warnings

File: news.rst [hunk 6]
```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
...
+currently existing code from IPython 3.x to the new various new packages
```
Comment (willingc) [inline news.rst]: Remove the first "new".

Add a period at the end of the sentence after the )

**Comment by willingc** [comment]
@Carreau, @takluyver caught most of the typos/grammar. I found a couple of others. I'll be on the train for a longgggg stretch tomorrow so feel free to send any docs stuff my way.

@takluyver I'm assuming British spellings are acceptable i.e. "organisation" :P

**Comment by takluyver** [comment]
> I'm assuming British spellings are acceptable i.e. "organisation" :P

Well, we don't call the language Americanese ;-)

**Comment by willingc** [comment]
:gb:

**Comment by minrk** [comment]
> Well, we don't call the language Americanese ;-)

```bash
$ locale
LANG="am_US.UTF-8"
```

**Comment by Carreau** [comment]
>Q:  How to you call someone who speak 3 languages : 

A: Trilingual

>Q:  How to you call someone who speak 2 languages : 

A: Bilingual

>Q:  How to you call someone who speak only 1 language : 

A: An American, and they don't even speak english.

**Comment by Carreau** [comment]
Comments taken into account.

**Comment by willingc** [comment]
+1 on the latest update. Enjoyed the jokes too.

**Pull request merged**